lib/parseBinaryGltf.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+var fs = require('fs');
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+var defaultValue = Cesium.defaultValue;
+var StringDecoder = require('string_decoder').StringDecoder;
+module.exports = parseBinaryGltf;
+
+function parseBinaryGltf(data) {
+    var sizeOfUint32 = Uint32Array.BYTES_PER_ELEMENT;
+    var byteOffset = 0;
+
+    if (data.slice(byteOffset, sizeOfUint32).toString() !== 'glTF') {
+        throw new Error('File is not valid binary glTF');
+    }
+
+    byteOffset += 3*sizeOfUint32;

```

Comment:
Instead of skipping over the version field, let's check it, and throw if it isn't version `1`.